styles/main.css

```diff
@@ -11,10 +11,9 @@ h2
 a
  {color: #2eb627 
  }
-.byline
- {opacity: 0.5;
-     color: #fffacd 
- }
+.byline {
+    color: rgba(255, 250, 205, 0.5); 
+}

```

Comment:
what you had works ( no marks lost), but I wanted you to use the rgba syntax for transparent colours, as we learned in class